app/models/user.rb

```diff
@@ -160,6 +158,14 @@ def update_access_token
 
   private
 
+  def password_complexity
+    return if google_oauth2_enabled?
+    return if password.blank? || (password =~ PWD_REGEX)
```

Comment:
can we extract this boolean into a separate method for readability?

Reply:
@jm96441n I think it is okay the way it is since it is not a complex condition

Reply:
that's true, but in terms of readability I think it's easier to read:
```
return if valid_password?
```
rather than
```
return if password.blank? || (password =~ PWD_REGEX)
```
Uncle Bob recommends sticking just about every boolean statement that's part of a conditional into it's own method in the book `Clean Code` just for the sake of making code easier to read
https://simpleprogrammer.com/refactoring-boolean-conditions-into-methods/